modules/aws-emr-cluster/main.tf

```diff
@@ -16,11 +16,8 @@ resource "aws_emr_cluster" "emr-cluster" {
 
   ec2_attributes {
     subnet_id                         = var.subnet_id
-    emr_managed_master_security_group = var.emr_managed_master_sg_id
-    additional_master_security_groups = var.emr_additional_master_sg_id
-    emr_managed_slave_security_group  = var.emr_managed_core_sg_id
-    additional_slave_security_groups  = var.emr_additional_core_sg_id
```

Comment:
Given that AWS only allows 4 additional security groups, should we continue to use `emr_managed_master_security_group` and `emr_managed_slave_security_group` for the security group to let the user pass in all 4 allowed in the additional?

Or does AWS allow you to specify 5 in this list if you don't specify that other field?

https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-additional-sec-groups.html